f5_openstack_agent/lbaasv2/drivers/bigip/lbaas_builder.py

```diff
@@ -272,7 +272,11 @@ def _assure_members(self, service, all_subnet_hints):
                         )
                         raise f5_ex.MemberCreationException(err.message)
                     else:
-                        self.pool_builder.update_member(svc, bigips)
+                        try:
+                            self.pool_builder.update_member(svc, bigips)
+                        except Exception as err:
```

Comment:
Could we create a unit test to ensure that the exception is raised appropriately and the provisioning status is set correctly?